### DISCUSSION TITLE
How import all images from folder

### DISCUSSION THREAD

**Discussion by jl0nd0n0** [parcel-bundler/parcel#5227] [Q&A]
Hi, Please.

How import all images from folder. 

I try

import foo from '../images/history/*.jpg';

error -> @parcel/core: Failed to resolve '../images/history/*.jpg' from './src/assets/components/nata-ui-article.js'

Thanks in advance

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/issues/4683
